tests/sanity/tests/recruiting/skills.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { test } from '@playwright/test'
+import { generateId, PlatformSetting, PlatformURI } from '../utils'
+
+test.use({
+  storageState: PlatformSetting
+})
+
+test.describe('skill tests', () => {
+  test.beforeEach(async ({ page }) => {
+    // Create user and workspace
+    await (await page.goto(`${PlatformURI}/workbench/sanity-ws/recruit`))?.finished()
+  })
+
+  test('create skill', async ({ page }) => {
```

Comment:
Please skip add tests for skills, since they should be reworked.

Reply:
done